doc/sphinx-guides/source/qa/overview.md

```diff
@@ -0,0 +1,46 @@
+# Overview
+
+```{contents} Contents:
+:local: 
+:depth: 3
+```
+
+## Introduction
+
+This guide describes the testing process used by QA at IQSS and provides a reference for others filling in for that role. Please note that many variations are possible, and the main thing is to catch bugs and provide a good quality product to the user community.
+
+## Workflow
+
+The basic workflow is as follows. Bugs or feature requests are submitted to GitHub by the community or by team members as [issues](https://github.com/IQSS/dataverse/issues). These issues are prioritized and added to a two-week sprint that is reflected on the GitHub {ref}`kanban-board`. As developers work on these issues, a GitHub branch is produced, code is contributed, and a pull request is made to merge these new changes back into the common {ref}`develop branch <develop-branch>` and ultimately released as part of the product.
```

Comment:
Lots of passive voice here. can we say, "Members of the Dataverse community or the development team submit bugs or feature requests to GitHub as issues." 
(also ":" after "as follows:" instead of "."?)

"Developers work on these issues in a GitHub branch where they make coding changes. Once they complete work on the issue they make a pull request to merge the changes back into the common 'develop branch'. Periodically a decision is made by the product manager with input from the architect and developers to release a new version of the software."